beep/protocol/maccor_to_biologic_mb.py

```diff
@@ -149,7 +149,7 @@ def _convert_time(self, time_str):
             return "{:.3f}".format(total_time_sec), "s"
 
     def _proc_step_to_seq(
-        self, proc_step, step_num, seq_from_step_num, goto_lower_bound, end_step_num
+        self, proc_step, step_num, seq_from_step_num, goto_lower_bound, end_step_num, current_range='10 A'
```

Comment:
Just to check, is the `current_range` make use of any context of the step creation here, or could we have just changed the `set_global_fields` helper for the conversion like so?:
```
def set_global_fields(seqs):
    for seq in seqs:
        seq["E range min (V)"] = "2.900"
        seq["E range max (V)"] = "4.300"
        seq["I Range"] = "10 A"
```
I'm trying to figure out if there's some context you're passing down to the converter that's specific to only _some_ steps/seqs and could not be applied to _all_ steps/seqs